Index: app/test/Ian/route.ts
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>export async function GET(request: Request) {\r\n    return Response.json({message: \"Hi I'm Ian!\"});\r\n}
===================================================================
diff --git a/app/test/Ian/route.ts b/app/test/Ian/route.ts
--- a/app/test/Ian/route.ts	(revision bf04b239f0071f33df46fdfbad8615c6b766dc63)
+++ b/app/test/Ian/route.ts	(date 1758686316392)
@@ -1,3 +1,3 @@
 export async function GET(request: Request) {
-    return Response.json({message: "Hi I'm Ian!"});
+    return Response.json({message:"Hi I'm Ian!"});
 }
\ No newline at end of file
